### ISSUE TITLE
Precision 3650 Tower Windows 11 Driver Pack

### ISSUE THREAD

**Issue by Aron-Love** [maurice-daly/DriverAutomationTool#544]
The tool is trying to download an A09 driver package that does not exist. According to Dell's Driver Pack site, the A08 pack release on May 16th is the latest: https://www.dell.com/support/kbdoc/en-us/000193920/precision-3650-tower-windows-11-driver-pack

When this happens, BITS gets stuck repeating "Error Error Error Error Error" in the log. I left the tool running last week, and this morning, it was hung up, and the log file was over 3.5 GB in size. Can we force downloads to fail so it moves on to the next one on the list?

**Comment by MikhailCompo** [comment]
Agreed. When the tool is unreliable, there needs to be better acceptance of this and failover mechanism incorporated.
